.github/workflows/tox-pytest.yml

```diff
@@ -166,13 +168,20 @@ jobs:
           service_account: "tox-pytest-github-action@catalyst-cooperative-pudl.iam.gserviceaccount.com"
 
       - name: Run ferc_to_sqlite
-        run: tox -e github_ferc_to_sqlite
-
+        run: | 
+          coverage --concurrency=multiprocessing run \
+            src/pudl/ferc_to_sqlite/cli.py --clobber ${{ env.ETL_COMMANDLINE_OPTIONS }} ${{ env.ETL_CONFIG }}
```

Comment:
Previously, Tox was responsible for installing all of the software dependencies, so everything that's not part of the `test-environment.yml` (which is almost everything) will need to be installed separately somehow if Tox isn't being used.  On the `conda-lockfile` branch that's done using `micromamba` and the locked dependencies.

Reply:
Yeah, this is now installed explicitly via `pip install .[test,datasette]` command above.